build/scripts/cibuild.ps1

```diff
@@ -52,6 +52,24 @@ function Terminate-BuildProcesses() {
     Get-Process vbcscompiler -ErrorAction SilentlyContinue | kill
 }
 
+# Ensure that procdump is available on the machine.  Returns the path to the directory that contains 
+# the procdump binaries (both 32 and 64 bit)
+function Ensure-ProcDump() {
```

Comment:
FYI. The Jenkins machines *should* have the entire SysInternals suite under `C:\SysInternals`